tensorflow/python/ops/array_ops.py

```diff
@@ -1085,6 +1085,7 @@ def boolean_mask(tensor, mask, name="boolean_mask"):
     tensor:  N-D tensor.
     mask:  K-D boolean tensor, K <= N and K must be known statically.
     name:  A name for this operation (optional).
+    axis: The axis in `tensor` to mask from. Defaults to the first dimension.
```

Comment:
You should clarify the documentation. In particular you should mention that it must be an integer, and please update lines 1080-1082. Ideally in addition `axis` would also be allowed to be a list of length K to be consistent with other `axis` arguments in the API, e.g. https://www.tensorflow.org/api_docs/python/tf/reduce_sum .